modules/ROOT/pages/sync/data_sources.adoc

```diff
@@ -0,0 +1,58 @@
+include::{partialsdir}/attributes.adoc[]
+
+//':context:' is a vital parameter. See: http://asciidoctor.org/docs/user-manual/#include-multiple
+:context: data_sources
+
+[id='{context}_data_sources']
+= Managing your data sources using the {sync-service} UI
+
+== Data Source Introduction
+
+A Data Source is a representation of your data storage solution. The Data Source consists of a *Name*, a *Type*, and a *Configuration*. The *Type* of Data Source which are currently supported are *Postgres* and *InMemory*. For information on how to use *Postgres*, refer to the documentation https://www.postgresql.org/docs[here]. The *InMemory* Data Source that is currently supported is *NeDB*, for more information on how to use this please refer to the the documentation https://github.com/louischatriot/nedb[here]. *Note* that the *InMemory* Data Source is not recommended for production use.
+
+
+
+== Data Source Options
+
+The data sources tab on the {sync-service} UI allows you to perform the following operatations:
+
+    * xref:data_sources#view-existing-data-sources[View all existing data sources]
+    * xref:data_sources#edit-an-existing-data-source[Edit an existing data source]
+    * xref:data_sources#delete-an-existing-data-source[Delete an existing data source]
+    * xref:data_sources#create-a-new-data-source[Create a new data source]
+
+=== View Existing Data Sources
+
+This is the default view on the {sync-service} UI. It lists all of the existing data sources. This view currently displays the data source name and the type of the data source. 
+
+image::datasourcesList.png[List Data Sources]
+
+Data sources can be filtered using the *name* property.
+
+image::filter.png[Filter by name]
+
+=== Edit an Existing Data Source
+
+To edit an existing data source, choose the *Edit Data Source* option from the data source menu.
+
+image::datasourceOptions.png[Choose Edit Opion]
+
+Update the data source with the new values.
+
+image::updateDatasource.png[Update Data Source]
+
+=== Delete an Existing Data Source
+
+To delete a data source, choose the *Delete Data Source* option from the data source menu. Next confirm that you wish to delete the data source.
```

Comment:
Let's add here that when you delete a data source all resolvers that are attached to it will also be deleted.

Reply:
Maybe as a `NOTE:`